rgw/v2/tests/s3_swift/test_cloud_transition.py

```diff
@@ -6,6 +6,7 @@
 
 <input_yaml>:
     configs/test_cloud_transitions.yaml
+    configs/test_cloud_transition_encrypted.yaml
```

Comment:
please update all other config files in <input_yaml>:

Reply:
Done